Documentation/observability/hubble-exporter.rst

```diff
@@ -151,3 +151,86 @@ The following is a complete example of configuring Hubble Exporter.
 
        {"flow":{"time":"2023-08-21T12:12:13.517394084Z","verdict":"DROPPED","IP":{"source":"fe80::64d8:8aff:fe72:fc14","destination":"ff02::2","ipVersion":"IPv6"},"l4":{"ICMPv6":{"type":133}},"source":{},"destination":{},"node_name":"kind-kind/kind-worker","drop_reason_desc":"INVALID_SOURCE_IP"},"node_name":"kind-kind/kind-worker","time":"2023-08-21T12:12:13.517394084Z"}
        {"flow":{"time":"2023-08-21T12:12:18.510175415Z","verdict":"DROPPED","IP":{"source":"10.244.1.60","destination":"10.244.1.5","ipVersion":"IPv4"},"l4":{"TCP":{"source_port":44916,"destination_port":80,"flags":{"SYN":true}}},"source":{"namespace":"default","pod_name":"xwing"},"destination":{"namespace":"default","pod_name":"deathstar-7848d6c4d5-th9v2"},"node_name":"kind-kind/kind-worker","drop_reason_desc":"POLICY_DENIED"},"node_name":"kind-kind/kind-worker","time":"2023-08-21T12:12:18.510175415Z"}
+
+
+Dynamic exporter configuration
+==============================
+
+Standard hubble exporter configuration accepts only one set of filters and
+requires cilium pod restart to change config. Dynamic flow logs allow configuring
+multiple filters at the same time and saving output in separate files.
+Additionally it does not require cilium pod restarts to apply changed configuration.
+
+**Dynamic Hubble Exporter** is enabled with Config Map property. It is disabled
+until you set a file path value for ``hubble-flowlogs-config-path``.
+
+.. include:: ../installation/k8s-install-download-release.rst
```

Comment:
I find the docs flow on this page a bit awkward - first we go through the basic exporter configuration, then filters and field masks, then a promise that dynamic exporters don't require an agent restart, and then all of the sudden I'm asked to download the Cilium tarball and install Cilium from scratch.

I think using Helm commands on this page is good but they should be used consistently within the whole page. Since this PR also adds Helm values for the basic exporter I believe that's doable - could you update the existing docs to use Helm commands too? Alternatively, IMO using `kubectl` commands is acceptable, or you can add a note assuming that Cilium is installed with Helm. Just these instructions to install Cilium out of the blue bother me, I would like to get rid of them.